cmd/root.go

```
@@ -190,15 +193,15 @@ Use "{{.CommandPath}} [command] --help" for more information about a command.{{e
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Version information",
-	Run: func(cmd *cobra.Command, args []string) {
-		displayVersionInfo()
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return displayVersionInfo()
 	},
 }
 
-func displayVersionInfo() {
-	fmt.Println("Version:", version)
-	fmt.Println("Build:", build)
-	os.Exit(0)
+func displayVersionInfo() error {
```

This will always return nil, so maybe it doesn't need to return an error?